hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/AvroConversionUtils.scala

```diff
@@ -242,4 +243,57 @@ object AvroConversionUtils {
     val nameParts = qualifiedName.split('.')
     (nameParts.last, nameParts.init.mkString("."))
   }
+
+  def areSchemasEquivalent(thisSchema: Schema, thatSchema: Schema): Boolean = {
```

Comment:
Why can't we simply do `thisSchema.equals(thatSchema)`? Please also add some javadoc if this equivalent check method does something special.

Reply:
Removed the code. It was redundant change.